ui/src/external/monaco.flux.autogrow.ts

```diff
@@ -0,0 +1,39 @@
+const MIN_HEIGHT = 100
```

Comment:
Why make this global?

Reply:
not global, it's scoped to the module instance. usually if you chose an arbitrary number in your code, you can expect it to change for arbitrary reasons. pulling it out as a constant at the top of a module signals "this is a compile time configuration" and makes it easier to change when the time comes without searching through all the app logic thinking "which one of these `100`s corresponded to that change?" and consolidating them in to logical groups if you have multiple instances